admin_guide/manage_nodes.adoc

```diff
@@ -523,7 +523,7 @@ During installation, {product-title} creates a configmap in the *openshift-node*
 To make configuration changes to an existing node, edit the appropriate configuration map. A _sync pod_ on each node
 watches for changes in the configuration maps. During installation, the sync pods are created by using _sync Daemonsets_,
 and a *_/etc/origin/node/node-config.yaml_* file, where the node configuration parameters reside, is added to each node. When a sync pod
-detects configuration map change, it updates the *_node-config.yaml_* on all nodes in that node group and restarts the appropriate nodes.
+detects configuration map change, it updates the *_node-config.yaml_* on all nodes in that node group and restarts the appropriate nodes `atomic-openshift-node.service`.
```

Comment:
Does the service restart on all of the nodes in the node group? If so, I'd say "it both updates the \*_node-config.yaml\_* file and restarts the \`atomic-openshift-node.service\` process on all nodes in that node group.

If it doesn't, I'd s/restarts the appropriate nodes \`atomic-openshift-node.service\`/restarts the \`atomic-openshift-node.service\` process on the nodes that require it.

If we know exactly which subset of nodes need to be restarted, I might replace  "that require it" with a description of those nodes.

Reply:
> Does the service restart on all of the nodes in the node group?

@kalexand-rh  Yes, it does. So I think "it both updates the *_node-config.yaml_* file and restarts the `atomic-openshift-node.service` process on all nodes in that node group." is appropriate on it.

Reply:
@bysnupy, is updating a node group configmap different than updating the node-config.yaml file?

Reply:
@kalexand-rh 
> is updating a node group configmap different than updating the node-config.yaml file?

It's same each other, but we warn not to update node-config.yaml directly due to overwriting as node group configmap.

1. update a node group configmap
2. updated node-config.yaml  as changed configurations by a sync pod
3. restarted atomic-openshift-node.service